policy_sentry/command/write_policy.py

```diff
@@ -12,21 +12,26 @@
 
 logger = logging.getLogger(__name__)
 
+
```

Comment:
The changes in this file only relate to spacing. I hit "reformat file" to make the IDE happy